lib/compat/wordpress-6.6/blocks.php

```diff
@@ -31,11 +31,27 @@ function gutenberg_replace_pattern_override_default_binding( $parsed_block ) {
 		// Build an binding array of all supported attributes.
 		// Note that this also omits the `__default` attribute from the
 		// resulting array.
-		foreach ( $supported_block_attrs[ $parsed_block['blockName'] ] as $attribute_name ) {
-			// Retain any non-pattern override bindings that might be present.
-			$updated_bindings[ $attribute_name ] = isset( $bindings[ $attribute_name ] )
-				? $bindings[ $attribute_name ]
-				: array( 'source' => 'core/pattern-overrides' );
```

Comment:
This code fires an error for the custom block type. I'm unsure of the best way to handle this, so I just copied the code from core for now to avoid the error while testing.